src/backbone.virtual-collection.js

```diff
@@ -71,23 +72,32 @@ var VirtualCollection = Backbone.VirtualCollection = Backbone.Collection.extend(
       this.listenTo(this.collection, eventName, _.partial(this.trigger, eventName));
     }, this));
   },
+  
+  _clearCache: function(){
+    this._changeCache = {
+      added: [],
+      removed: [],
+      merged: []
+    };
+  },
 
   _onUpdate: function (collection, options) {
-    this.trigger('update', this, options);
+    var newOptions = _.extend({}, options, {changes: this._changeCache});    
```

Comment:
yep, here is a reference to the diff
https://github.com/jashkenas/backbone/compare/1.2.3...1.3.3#diff-0d56d0d310de7ff18b3cef9c2f8f75dcR926